frontend/src/components/DocumentViewer/DocumentViewBox.css

```diff
@@ -26,27 +26,46 @@
 .document-popup-inner {
 	display: flex;
 	flex-direction: column;
+	justify-content: center;
+	height: 85vh;
 }
 
 .view-documents-main {
 	position: relative;
 	display: flex;
+	flex-grow: 1;
 	flex-direction: column;
+	overflow: hidden;
 	box-sizing: border-box;
-	height: 75vh;
-	padding: 2rem;
+	padding: 2rem 3rem;
 	border: 0.1875rem solid var(--main-border-colour);
 	background: var(--main-background);
 	color: var(--main-text-colour);
-	aspect-ratio: 7/8;
 }
 
 .document-viewer-container {
 	flex-grow: 1;
 	overflow-y: auto;
+	width: 32.5rem;
```

Comment:
This is to keep the paragraph width consistent in terms of characters per line, regardless of zoom / font size. There could be other approaches, but in any case the fixed aspect ratio wasn't working well.